<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSpriteDialog</class>
 <widget class="QDialog" name="EditSpriteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sprite Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QMenuBar" name="menuBar">
     <widget class="QMenu" name="menuFile">
      <property name="title">
       <string>&amp;File</string>
      </property>
      <addaction name="actionNew"/>
      <addaction name="actionCreateFromFile"/>
      <addaction name="actionAddFromFile"/>
      <addaction name="separator"/>
      <addaction name="actionSaveAsPngFile"/>
      <addaction name="separator"/>
      <addaction name="actionCreateFromStrip"/>
      <addaction name="actionAddFromStrip"/>
      <addaction name="separator"/>
      <addaction name="actionCloseSavingChanges"/>
     </widget>
     <widget class="QMenu" name="menuEdit">
      <property name="title">
       <string>&amp;Edit</string>
      </property>
      <addaction name="actionUndo"/>
      <addaction name="actionRedo"/>
      <addaction name="separator"/>
      <addaction name="actionCut"/>
      <addaction name="actionCopy"/>
      <addaction name="actionPaste"/>
      <addaction name="separator"/>
      <addaction name="actionErase"/>
      <addaction name="actionDelete"/>
      <addaction name="separator"/>
      <addaction name="actionMoveLeft"/>
      <addaction name="actionMoveRight"/>
      <addaction name="separator"/>
      <addaction name="actionAddEmpty"/>
      <addaction name="actionInsertEmpty"/>
      <addaction name="separator"/>
      <addaction name="actionEdit"/>
      <addaction name="separator"/>
      <addaction name="actionSetTransparencyBackground"/>
     </widget>
     <widget class="QMenu" name="menuTransform">
      <property name="title">
       <string>&amp;Transform</string>
      </property>
     </widget>
     <widget class="QMenu" name="menuImages">
      <property name="title">
       <string>&amp;Images</string>
      </property>
     </widget>
     <widget class="QMenu" name="menuAnimation">
      <property name="title">
       <string>&amp;Animation</string>
      </property>
     </widget>
     <addaction name="menuFile"/>
     <addaction name="menuEdit"/>
     <addaction name="menuTransform"/>
     <addaction name="menuImages"/>
     <addaction name="menuAnimation"/>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolBar" name="toolBar">
        <property name="movable">
         <bool>false</bool>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="floatable">
         <bool>false</bool>
        </property>
        <addaction name="actionCloseSavingChanges"/>
        <addaction name="separator"/>
        <addaction name="actionNew"/>
        <addaction name="actionCreateFromFile"/>
        <addaction name="actionAddFromFile"/>
        <addaction name="actionSaveAsPngFile"/>
        <addaction name="separator"/>
        <addaction name="actionInsertEmpty"/>
        <addaction name="actionAddEmpty"/>
        <addaction name="separator"/>
        <addaction name="actionUndo"/>
        <addaction name="actionRedo"/>
        <addaction name="separator"/>
        <addaction name="actionCut"/>
        <addaction name="actionCopy"/>
        <addaction name="actionPaste"/>
        <addaction name="separator"/>
        <addaction name="actionMoveLeft"/>
        <addaction name="actionMoveRight"/>
        <addaction name="separator"/>
        <addaction name="actionEdit"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBoxShowPreview">
         <property name="text">
          <string>Show Preview</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QListView" name="listView">
      <property name="spacing">
       <number>16</number>
      </property>
      <property name="viewMode">
       <enum>QListView::ViewMode::IconMode</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QStatusBar" name="statusBar"/>
   </item>
  </layout>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../resources_editor.qrc">
     <normaloff>:/qtgameengine/icons/new.png</normaloff>:/qtgameengine/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New...</string>
   </property>
   <property name="toolTip">
    <string>Create a new sprite</string>
   </property>
  </action>
  <action name="actionCreateFromFile">
   <property name="icon">
    <iconset resource="../resources_editor.qrc">
     <normaloff>:/qtgameengine/icons/open.png</normaloff>:/qtgameengine/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Create from file...</string>
   </property>
   <property name="toolTip">
    <string>Create new sprite from file</string>
   </property>
  </action>
  <action name="actionAddFromFile">
   <property name="icon">
    <iconset resource="../resources_editor.qrc">
     <normaloff>:/qtgameengine/icons/create-group.png</normaloff>:/qtgameengine/icons/create-group.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add from file...</string>
   </property>
   <property name="toolTip">
    <string>Add a sprite from file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSaveAsPngFile">
   <property name="icon">
    <iconset resource="../resources_editor.qrc">
     <normaloff>:/qtgameengine/icons/save.png</normaloff>:/qtgameengine/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save as PNG File...</string>
   </property>
   <property name="toolTip">
    <string>Save the sprite as PNG File</string>
   </property>
  </action>
  <action name="actionCreateFromStrip">
   <property name="text">
    <string>Create from Strip...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionAddFromStrip">
   <property name="text">
    <string>Add from Stri&amp;p...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionCloseSavingChanges">
   <property name="icon">
    <iconset resource="../resources_editor.qrc">
     <normaloff>:/qtgameengine/icons/ok.png</normaloff>:/qtgameengine/icons/ok.png</iconset>
   </property>
   <property name="text">
    <string>Close Saving Changes</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../resources_editor.qrc">
     <normaloff>:/qtgameengine/icons/undo.png</normaloff>:/qtgameengine/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../resources_editor.qrc">
     <normaloff>:/qtgameengine/icons/redo.png</normaloff>:/qtgameengine/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../resources_editor.qrc">
     <normaloff>:/qtgameengine/icons/cut.png</normaloff>:/qtgameengine/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../resources_editor.qrc">
     <normaloff>:/qtgameengine/icons/copy.png</normaloff>:/qtgameengine/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../resources_editor.qrc">
     <normaloff>:/qtgameengine/icons/paste.png</normaloff>:/qtgameengine/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionErase">
   <property name="text">
    <string>&amp;Erase...</string>
   </property>
   <property name="shortcut">
    <string>Shift+Del</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="actionMoveLeft">
   <property name="text">
    <string>Move &amp;Left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionMoveRight">
   <property name="text">
    <string>&amp;Move Right</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAddEmpty">
   <property name="text">
    <string>&amp;Add Empty</string>
   </property>
   <property name="shortcut">
    <string>Shift+A</string>
   </property>
  </action>
  <action name="actionInsertEmpty">
   <property name="text">
    <string>&amp;Insert Empty</string>
   </property>
   <property name="shortcut">
    <string>Shift+I</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSetTransparencyBackground">
   <property name="text">
    <string>&amp;Set Transparency Background...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources_editor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionCloseSavingChanges</sender>
   <signal>triggered()</signal>
   <receiver>EditSpriteDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
